### PR TITLE
Add Odoo icon

### PR DESCRIPTION
**Pull request by lucas-louis**
odoo

**Issue: #6961**
**Alexa rank: 1,429**
  

### Checklist
  - [X] I updated the JSON data in `_data/simple-icons.json`
  - [X] I optimized the icon with SVGO or SVGOMG
  - [X] The SVG `viewbox` is `0 0 24 24`

### Description

The color hex value is from: https://www.odoo.com/page/brand-assets
I optimized the lofo with SVGO

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -6787,6 +6787,12 @@
             "hex": "EE8208",
             "source": "https://insideok.ru/brandbook"
         },
+        {
+            "title": "Odoo",
+            "hex": "714B67",
+            "source": "https://www.odoo.com/fr_FR/page/brand-assets#part_6",
```
Comment (dirien) [inline _data/simple-icons.json:6793]: Can you enter just the root for the source?

**Review by dirien — DISMISSED** [review]
hi @lucas-louis,

thanks for this contribution.

I see some differences, maybe up the precision?
image


And i used as reference the original one from Odoo website.

https://odoocdn.com/openerp_website/static/src/img/2016/logos/odoo_logo.svg

**Review by LitoMore — DISMISSED** [review]
image

@adamrusted This looks not correct in the preview. Could you check those shapes?

**Comment by adamrusted** [comment]
Sure can. Bear with.

**Comment by LitoMore** [comment]
image

It's kind of weird, the icon width shown in Inkscape is not `24.000` but it passed our CI.

**Comment by adamrusted** [comment]
> It's kind of weird, the icon width shown in Inkscape is not `24.000` but it passed our CI.

Very odd. The compound paths in this one were really playing up for me...

File: icons/odoo.svg
Comment (PratyushJayachandran) [inline icons/odoo.svg:1]: I tried to recreate and ended up with the exact same SVG code.

**Review by PratyushJayachandran — APPROVED** [review]

File: icons/odoo.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Odoo</title><path d="M11.224 8.184a.576.576 0 0 0-.576.576v1.845a2.896 2.896 0 1 0 1.152 2.312V8.76a.576.576 0 0 0-.576-.576zm-2.323 6.443a1.708 1.708 0 1 1 .016-3.418 1.708 1.708 0 1 1-.016 3.418zm6.062-4.608a2.898 2.898 0 0 0-.008 5.795l.003.003a2.898 2.898 0 1 0 .005-5.798zm-.006 4.608a1.71 1.71 0 1 1 .005-3.42 1.71 1.71 0 0 1-.005 3.42zM2.899 10.013a2.9 2.9 0 1 0 0 5.798 2.9 2.9 0 0 0 0-5.798zm-.027 4.619a1.71 1.71 0 0 1-1.709-1.699v-.011a1.709 1.709 0 1 1 1.709 1.71zm18.232-4.613a2.898 2.898 0 1 0-.006 5.796 2.898 2.898 0 0 0 .006-5.796zm-.005 4.608a1.71 1.71 0 1 1 .005-3.42 1.71 1.71 0 0 1-.005 3.42z"/></svg>
```
Comment (PratyushJayachandran) [inline icons/odoo.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Odoo</title><path d="M2.9 10C1.3 10 0 11.3 0 12.9c0 1.6 1.3 2.9 2.9 2.9s2.9-1.3 2.9-2.9c0-1.6-1.3-2.9-2.9-2.9zm0 4.6c-.9 0-1.7-.8-1.7-1.7s.8-1.7 1.7-1.7 1.7.8 1.7 1.7-.8 1.7-1.7 1.7zm8.4-6.4c-.3 0-.6.3-.6.6v1.7c-.5-.4-1.1-.5-1.7-.5-1.7 0-2.9 1.2-2.9 2.9s1.3 2.9 2.9 2.9 2.9-1.3 2.9-2.9V8.8c0-.3-.3-.6-.6-.6zM9 14.6c-.9 0-1.7-.8-1.7-1.7S8 11.2 9 11.2s1.7.8 1.7 1.7-.8 1.7-1.7 1.7zM21.1 10c-1.6 0-2.9 1.3-2.9 2.9 0 1.6 1.3 2.9 2.9 2.9s2.9-1.3 2.9-2.9c0-1.6-1.3-2.9-2.9-2.9zm0 4.6c-.9 0-1.7-.8-1.7-1.7s.8-1.7 1.7-1.7 1.7.8 1.7 1.7-.8 1.7-1.7 1.7zM15 10c-1.6 0-2.9 1.3-2.9 2.9 0 1.6 1.3 2.9 2.9 2.9s2.9-1.3 2.9-2.9c0-1.6-1.3-2.9-2.9-2.9zm0 4.6c-.9 0-1.7-.8-1.7-1.7s.8-1.7 1.7-1.7 1.7.8 1.7 1.7-.7 1.7-1.7 1.7z"/></svg>
```

**Review by PratyushJayachandran — APPROVED** [review]

**Comment by PratyushJayachandran** [comment]
@adamrusted @LitoMore please re review the PR

**Review by adamrusted — DISMISSED** [review]
image
@PratyushJayachandran - could you take another look please?

File: icons/odoo.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Odoo</title><path d="M21.101 15.787A2.899 2.899 0 1 1 24 12.89c0 1.6-1.298 2.898-2.899 2.898zm0-1.198a1.7 1.7 0 1 0 0-3.4 1.7 1.7 0 0 0 0 3.4zm-6.067 1.198a2.899 2.899 0 1 1 2.898-2.898c0 1.6-1.297 2.898-2.898 2.898zm0-1.198a1.7 1.7 0 1 0 0-3.4 1.7 1.7 0 0 0 0 3.4zm-3.17-1.739c0 1.62-1.297 2.937-2.898 2.937s-2.898-1.314-2.898-2.934S7.246 9.99 8.966 9.99c.636 0 1.223.136 1.7.487V8.796a.6.6 0 0 1 1.199 0v4.054zm-2.898 1.74a1.7 1.7 0 1 0 0-3.402 1.7 1.7 0 0 0 0 3.401zM2.9 15.786a2.899 2.899 0 1 1 2.898-2.898c0 1.6-1.298 2.898-2.898 2.898zm0-1.198a1.7 1.7 0 1 0 0-3.4 1.7 1.7 0 0 0 0 3.4z"/></svg>
```
Comment (PratyushJayachandran) [inline icons/odoo.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Odoo</title><path d="M21.1 15.796a2.9 2.9 0 1 1 0-5.8 2.9 2.9 0 0 1 0 5.8zm0-1.2a1.7 1.7 0 1 0 .002-3.4H21.1a1.7 1.7 0 0 0 0 3.4zm-6.065 1.2a2.9 2.9 0 1 1 2.9-2.9c0 1.6-1.3 2.9-2.9 2.9zm0-1.2a1.7 1.7 0 0 0 .001-3.4h-.001a1.7 1.7 0 0 0 0 3.4zm-3.17-1.738c0 1.62-1.298 2.938-2.898 2.938s-2.9-1.314-2.9-2.936S7.247 10 8.967 10c.636 0 1.224.134 1.7.484v-1.68a.6.6 0 0 1 1.2 0v4.054h-.002zm-2.898 1.74a1.7 1.7 0 0 0 .002-3.4h-.002a1.7 1.7 0 0 0 0 3.4zM2.9 15.796a2.9 2.9 0 1 1 2.9-2.9c0 1.6-1.3 2.9-2.9 2.9zm0-1.2a1.7 1.7 0 0 0 .001-3.4H2.9a1.7 1.7 0 0 0 0 3.4z"/></svg>
```
Fix errors

**Comment by PratyushJayachandran** [comment]
> @PratyushJayachandran - could you take another look please?

I fixed the icon but running into that new line error. Please check the icon again.

**Review by adamrusted — CHANGES_REQUESTED** [review]
Looks like there's a precision issue here. The paths are a pretty close match, but only slightly out across the board. Could you regenerate using `npx svgo icons/odoo.svg --p 4` and see if that gives us a better outcome?

File: icons/odoo.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Odoo</title><path d="M21.1 15.796a2.9 2.9 0 1 1 0-5.8 2.9 2.9 0 0 1 0 5.8zm0-1.2a1.7 1.7 0 1 0 .002-3.4H21.1a1.7 1.7 0 0 0 0 3.4zm-6.065 1.2a2.9 2.9 0 1 1 2.9-2.9c0 1.6-1.3 2.9-2.9 2.9zm0-1.2a1.7 1.7 0 0 0 .001-3.4h-.001a1.7 1.7 0 0 0 0 3.4zm-3.17-1.738c0 1.62-1.298 2.938-2.898 2.938s-2.9-1.314-2.9-2.936S7.247 10 8.967 10c.636 0 1.224.134 1.7.484v-1.68a.6.6 0 0 1 1.2 0v4.054h-.002zm-2.898 1.74a1.7 1.7 0 0 0 .002-3.4h-.002a1.7 1.7 0 0 0 0 3.4zM2.9 15.796a2.9 2.9 0 1 1 2.9-2.9c0 1.6-1.3 2.9-2.9 2.9zm0-1.2a1.7 1.7 0 0 0 .001-3.4H2.9a1.7 1.7 0 0 0 0 3.4z"/></svg>
```
Comment (PratyushJayachandran) [inline icons/odoo.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Odoo</title><path d="M21.1002 15.7957c-1.6015 0-2.8997-1.2983-2.8997-2.8998s1.2983-2.8997 2.8997-2.8997c1.6015 0 2.8998 1.2982 2.8998 2.8997 0 1.5999-1.2979 2.8998-2.8998 2.8998zm0-1.2c.9388.0006 1.7003-.7601 1.7008-1.6989.0004-.9388-.7602-1.7003-1.699-1.7007h-.0018c-.9388.0004-1.6994.7619-1.699 1.7007.0005.9381.761 1.6985 1.699 1.699zm-6.0655 1.2c-1.6014 0-2.8997-1.2983-2.8997-2.8998s1.2983-2.8997 2.8997-2.8997c1.6015 0 2.8998 1.2982 2.8998 2.8997 0 1.5999-1.2999 2.8998-2.8998 2.8998zm0-1.2c.9389.0006 1.7003-.7601 1.7008-1.6989.0005-.9388-.7602-1.7003-1.699-1.7007h-.0018c-.9388.0004-1.6994.7619-1.699 1.7007.0005.9381.761 1.6985 1.699 1.699zM11.865 12.858c0 1.6199-1.2979 2.9378-2.8977 2.9378s-2.8998-1.314-2.8998-2.9358 1.1799-2.8597 2.8998-2.8597c.6359 0 1.2239.134 1.6998.484v-1.68a.6.6 0 0 1 1.2 0v4.0537h-.002zm-2.8977 1.7399c.9388.0005 1.7002-.7602 1.7007-1.699.0005-.9388-.7602-1.7003-1.699-1.7007h-.0017c-.9389.0004-1.6995.7619-1.699 1.7007.0004.9381.7608 1.6985 1.699 1.699zm-6.0675 1.1979C1.2983 15.7957 0 14.4974 0 12.8959s1.2983-2.8997 2.8998-2.8997 2.8997 1.2982 2.8997 2.8997c0 1.5999-1.2999 2.8998-2.8997 2.8998zm0-1.2c.9388.0006 1.7002-.7601 1.7007-1.699.0005-.9387-.7602-1.7002-1.699-1.7006h-.0017c-.9388.0004-1.6995.7619-1.699 1.7007.0004.9381.7608 1.6985 1.699 1.699z"/></svg>
```

**Comment by PratyushJayachandran** [comment]
> Looks like there's a precision issue here. The paths are a pretty close match, but only slightly out across the board. Could you regenerate using `npx svgo icons/odoo.svg --p 4` and see if that gives us a better outcome?

Done. Please check again :)

**Review by adamrusted — APPROVED** [review]

**Pull request merged**